changelog.txt

```diff
@@ -10,6 +10,7 @@
 * Tweak - Use wp_ajax prefix for its built-in security for Add Payment Method action
 * Dev - Fix WooCommerce version fetching in GitHub workflows
 * Dev - Fix failing test cases associated with WooCommerce 10.0.x
+* Fix - Fix PHP warning about undefined labels for custom checkout fields
```

Comment:
Given that we're also fixing the required error message, I think we should reflect that in the entry.
```suggestion
* Fix - Fix required field error message and PHP warning for custom checkout fields that don't have a label
```